example/pom.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Rename plz module `example` to `examples` since we already have `benchmarks`; the plural.